src/widget/wslidercomposed.cpp

```diff
@@ -150,23 +204,88 @@ void WSliderComposed::paintEvent(QPaintEvent * /*unused*/) {
     QStylePainter p(this);
     p.drawPrimitive(QStyle::PE_Widget, option);
 
+    // Painter for the level bar
+//    QPainter pl(this);
+
```

Comment:
Will remove this with the demo styles.

Reply:
![3xoj92](https://user-images.githubusercontent.com/1834516/80006227-e6459e00-84c4-11ea-914c-10a624740e4a.jpg)

Reply:
@ronso0 ping

Reply:
Aahm, yes, thanks for reminding me of my reminder..